src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -153,14 +174,18 @@ public void pairDevice(
    * @param setupPincode the pincode for the device
    * @param csrNonce the 32-byte CSR nonce to use, or null if we want to use an internally randomly
    *     generated CSR nonce.
+   * @param networkCredentials the credentials (Wi-Fi or Thread) to be provisioned
+   * @param icdRegistrationInfo the informations for ICD registration. For detailed information
```

Comment:
i would prefer indicating how to use this optional icdRegistrationInfo. if exists, we would not locally generate the random key.

Reply:
update doxygen. please check about this.

Reply:
thanks